templates/webapi/test/infopanel.html.ep

```diff
@@ -123,6 +123,7 @@
                     % if ($clone_of) {
                         Clone of
                         %= link_to $clone_of->id => url_for ('test', testid => $clone_of->id)
+                        (restarted <%= $ancestors >= $ancestors_limit ? 'at least ' : '' %><%= $ancestors %> time<%= $ancestors != 1 ? 's' : '' %>)
```

Comment:
`$ancestors` will never actually be greater than `$ancestors_limit` and `$ancestors` is also what's being displayed. So in order for "more than" to make sense I needed to subtract one from `$ancestors` in case of `$ancestors >= $ancestors_limit`. Not sure whether I like that, what's wrong with "at least"?

Reply:
Well, "at least" sounds like we might want to motivate for people to restart more when what we commonly want is to motivate for is to limit the number of restarts

Reply:
I could also just put a `>=` sign in there. I suppose a mathematical symbol is quite neutral in that regard.

Reply:
I like your current version with "more than"